core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -5260,17 +5260,19 @@ private static void computeVersion(ServletContext context) {
      * Grants ability to configure any and all aspects of the Jenkins instance
      */
     public static final Permission ADMINISTER = Permission.HUDSON_ADMINISTER;
-    public static final Permission READ = new Permission(PERMISSIONS,"Read",Messages._Hudson_ReadPermission_Description(),Permission.READ,PermissionScope.JENKINS);
-    @Deprecated
-    /** @deprecated as of TODO use {@link Jenkins#ADMINISTER} */
-    @Restricted(NoExternalUse.class)
-    public static final Permission RUN_SCRIPTS = new Permission(PERMISSIONS, "RunScripts", Messages._Hudson_RunScriptsPermission_Description(),ADMINISTER,PermissionScope.JENKINS);
 
     /**
      * Allows non-privilege escalating configuration permission for a Jenkins instance.  Actions which could result
      * in a privilege  escalation (such as RUN_SCRIPTS) require explicit ADMINISTER permission
      */
     public static final Permission CONFIGURE_JENKINS = new Permission(PERMISSIONS, "Configure", Messages._Hudson_ConfigureJenkins_Description(),ADMINISTER, PermissionScope.JENKINS);
+    
+    public static final Permission READ = new Permission(PERMISSIONS,"Read",Messages._Hudson_ReadPermission_Description(),CONFIGURE_JENKINS,PermissionScope.JENKINS);
```

Comment:
this means that you can now have `Permission.READ` but you won't get `Jenkins.READ` (which is a bit funky)..

Hmm.... is there a generic `CONFIGURE` that this can imply?  (that can itself be used to imply the generic read?)

Reply:
I think adding a generic `READ` makes sense